blog/_posts/2017-02-22-repl-0.6-highlights.md

```diff
@@ -0,0 +1,102 @@
+
+@time f();
+```
+
+which could then be pasted back into the REPL to be executed.
+In 0.6, the REPL comes with a feature called *prompt pasting* by which a pasted expression starting with `julia> ` automatically has prompts and eventual output scrubbed before being executed.
+If you have a 0.6 REPL currently at hand, you can try to copy and paste the example with the prompt and output above directly into the REPL and you will see that it works in exactly the same way as if we had manually removed them.
+
+Unfortunately, for technical reasons (no support for bracket pasting), this feature does not work in the standard cmd-terminal on Windows.
+There are however multiple alternative terminal emulators on Windows that do support this, e.g. [Console2](https://sourceforge.net/projects/console/) or [Cygwin](https://www.cygwin.com/).
+
+## Printing of stacktraces
+
+When an error occurs, the REPL prints the exception that was thrown along with a [stack trace](https://en.wikipedia.org/wiki/Stack_trace): a list tracing back through the sequence of function calls and source-code lines that were being executed when the error occurred. In previous versions of Julia, however, the stack trace might omit some function calls that were [inlined](https://en.wikipedia.org/wiki/Inline_expansion), making it harder to understand. In Julia 0.6, however, the stack trace includes function calls that were inlined. Unfortunately, in combination with the Julia idiom of breaking complex tasks into many smaller functions, this can sometimes lead to long stack traces. Furthermore, the stack trace includes the type signature of the methods being called, but these type signatures can sometime be quite verbose (e.g. for union types or types taking many parameters). Julia 0.6 greatly improves the readability of this output by coloration, numbering, and interactive features described below.
+
+In earlier julia versions, stacktraces were being printed without much structure.
+Everything was printed in bold red which made it a bit difficult to find the information one was looking for.
+As an example, here is a screenshot from an error thrown on 0.5:
+
+![Stacktrace in 0.5](/images/blog/2017-02-24-repl0.6/0.5stacktrace.png)
+
+Finding the relevant information can be quite difficult, especially the file name and line numbers since they are not aligned in any way.
+In 0.6, this is what is currently shown for the same error:
+
+![Stacktrace in 0.6](/images/blog/2017-02-24-repl0.6/0.6stacktrace.png)
+
+We stopped printing the contents of the `Array` on a `BoundsError` but more relevant for this section, the way stacktraces are shown is significantly different.
+They are no longer written in bold red, which helps distinguish it from the error message and the parts that are most likely to be interesting (the function name and the line info) are now bold, making them stick out from the surrounding text.
+Every stackframe is also numbered which makes them easier to refer to.
```

Comment:
Perhaps "Stacktrace presentation has improved significantly: Stacktraces no longer print in red, while associated error messages continue to print in red. Stacktraces also no longer print entirely in bold; only function names and line information now appear in bold. These changes visually separate the most-frequently sought information (the associated error message, and function names and line information) from the surrounding text, making that information much easier to pick out. Additionally, each stackframe in the stacktrace prints numbered, making individual stackframes easier to refer to. (Apart from these changes to stacktrace presentation, `BoundsError`s no longer display with the associated collection's contents, decreasing visual noise.)"?